src/main/java/uk/gov/companieshouse/acsp/controllers/AcspController.java

```diff
@@ -43,7 +43,7 @@ public ResponseEntity<Object> getAcspData(@PathVariable(TRANSACTION_ID_KEY) Stri
         if (acspData == null){
```

Comment:
This should check for .isEmpty() instead of null

Reply:
addressed